include/igraph_error.h

```diff
@@ -217,7 +217,7 @@ typedef void igraph_error_handler_t (const char * reason, const char * file,
  * program.
  */
 
-extern igraph_error_handler_t igraph_error_handler_abort;
+DECLDIR igraph_error_handler_t igraph_error_handler_abort;
```

Comment:
@ntamas , please note that these `extern`s are not necessary. They seem to define a symbol, but they actually define a function, and if my understanding is correct `extern` is unnecessary there (it is already implied).

Reply:
Yes, it seems to compile this way so all is good.